### ISSUE TITLE
[BUG] Transaction description fix

### ISSUE THREAD

**Issue by fckt** [near/docs#254]
**Describe the bug**
Transaction description is not correct
**Expected behavior**

```
Transactions

A Transaction is a collection of Actions augmented with critical information about their

origin (ie. cryptographically signed by signer)
destination or intention (ie. sent or applied to receiver)
recency (ie. block_hash distance from most recent block is within acceptable limits)
uniqueness (ie. nonce must be unique for a given signer's AccessKey)
```

A list of actions in transaction describes what should be done at the destination (`receiver_id` account.

**:pencil: Location**
https://docs.nearprotocol.com/docs/concepts/transaction#transactions

**Comment by amgando** [comment]
> _destination or intention (ie. sent or applied to receiver)_

i think this is what i wrote here, but obviously it's not so clear.  will try to fix.

how about this...I added your suggestions in **bold**

> Transactions
> 
> A Transaction is a collection of Actions.  **Together, the actions describe _what should be done at the destination_ (receiver_id account).**  The transaction also holds other critical information, metadata, about the list of actions:
> 
> - origin (ie. cryptographically signed by signer)
> - destination or intention (ie. sent or applied to receiver), **_as mentioned above_**
> - recency (ie. block_hash distance from most recent block is within acceptable limits)
> - uniqueness (ie. nonce must be unique for a given signer's AccessKey)
>

**Comment by fckt** [comment]
A difference between Transaction and a Receipt. 

The main difference is that we do not apply Transaction, but we do apply Receipt. When runtime process transactions (verified transactions - during the chunk application) we only charge Transaction (subtract a deposit and gas) for actions which will be applied in the `destination` and generate Receipt with these actions which will be then applied. A good mental model is to think that **a Receipt - is a paid message to be executed on a destination**.

> A Transaction is a collection of Actions. Together, the actions describe what should be done at the destination (receiver_id account). The transaction also holds other critical information, metadata, about the list of actions:

In part, a source of the confusion lies in the following:  to think that the `actions` in transaction (receipt) are not mutually dependent. In reality it's not a case: action list must be treated as a the whole thing. Some combinations (and order) of actions have no sense. So, **an action is just a unit of composition**.

There is a bunch of administrative actions which requires `sender` to be `receiver` (Transaction or a Receipt which was being sent to **self**). They are: `Action::DeployContract`, `Action::Stake`, `Action::AddKey`, `Action::DeleteKey` and `Action::DeleteAccount`. `Action::CreateAccount` grants permission to evaluate subsequent actions like the `sender` is the `receiver` (he sends a message to itself). `Action::DeleteAccount` sets it back.

**Comment by amgando** [comment]
fixed in #378 but not closing automatically, not sure why
